src/github/controller.py

```diff
@@ -53,15 +53,11 @@ def upsert_comment(pull_request: PullRequest, comment: Comment):
     pull_request_id = pull_request.id()
     task_id = dynamodb_client.get_asana_id_from_github_node_id(pull_request_id)
     if task_id is None:
-        logger.info(
-            f"Task not found for pull request {pull_request_id}. Running a full sync!"
+        logger.error(
+            f"Task not found for pull request {pull_request_id}. Exiting!"
         )
-        # TODO: Full sync
```

Comment:
doing a full sync is kind of difficult at this point in the code so instead of logging that we're doing something we actually aren't, I'm just logging an error. This doesn't seem to happen that often

Reply:
I'm a little worried about not syncing tasks. I feel like a complaint people have is not having data from a PR. Would exiting here mean there can be cases where a PR could exist without an associated task?

Reply:
There are definitely instances where a PR could exist without an associated task but full syncing was never properly implemented so there's no functional difference (i.e. this has always been an issue).

We do want some kind of full upsert on a task if the task id does not exist but that will have to come later